### ISSUE TITLE
Actions > Load RDF page accessibility issues

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4027]
On the Load RDF page, the "Load RDF into Editor" should be an H1. When the Submit button is disabled, it should state that it is disabled. It currently just says it is 'dimmed' in the Voiceover.
